src/runtime/storage/searchable-store.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * @license
+ * Copyright 2020 Google LLC.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * Code distributed by Google as part of this project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+import {Type} from '../type.js';
+import {Manifest} from '../manifest.js';
+import {ToStore} from './storage.js';
+import {AbstractStore} from './abstract-store.js';
+
+export interface SearchableStore {
+  context: Manifest;
```

Comment:
Is this needed? It's quite a big constraint to require a manifest to be a part of the StoreContext.

Reply:
Just realized: `Manifest` basically implements the store context contract. Thus, I can keep the context, but as a `StoreContext`.